lib/paperclip/gd_image_processor.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Paperclip
+  class GDImageProcessor < Processor
+    def make
+      original_geometry = Geometry.new(*FastImage.size(file))
+      computed_geometry = original_geometry.resize_to(target_geometry_str)
+      file.rewind
+      original_geometry.to_s[/\d+x\d+/] == computed_geometry.to_s[/\d+x\d+/] ? temp_copy(file) : FastImage.resize(file, computed_geometry.width, computed_geometry.height)
```

Comment:
This line got big. Aren't we using some linter to standardise some style guides?

Reply:
We've been using long lines in porta. We can discuss with team changing our style if you see issues with that.

Reply:
I agree to bring this kind of code style discussions to the team. After checking some of the code in this repo, I can see only improvements coming from a discussed/agreed style guide.